public/css/index.css

```
@@ -0,0 +1,332 @@
+@charset "utf-8"
+*{
+	margin: 0;
+	padding: 0;
+	outline: none;
+  box-sizing: inherit;
+}
+html {
+  height: 100%;
+  font-size: 1vh;
+}
+body {
+	width: 100%;
+	height: 100%;
+  background-size: unset;
+	font-family: "Segoe UI", san-serif;
+	line-height: 100%;
+  box-sizing: border-box;
+  background: black;
+}
+header {
+  float: left;
+  width: 100%;
+  height: 13%;
+  box-sizing: border-box;
+}
+
+.dropbtn {
+  background-color: black;
+  color: white;
+  font-size: 1.5vw;
+  border: none;
+  border-radius: 10rem;
+  width: 7vw;
+  height:4vh;
+	bottom: 0;
+  font-family: gotham_black;
+}
+
+.about {
+  position: absolute;
+  display: inline-block;
+  margin-left: 22vw;
+  margin-top: 4vh;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+	min-width: 7vw;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+  font-size: 1.5vh;
+}
+
+.acc,.profile,.exit {
+  color: black;
+  padding: 10px;
+	text-decoration: none;
+  display: block;
+  font-family: gotham_black;
+
+}
+
+.dropdown-content a:hover {
```

Тут можно сделать проще
```suggestion
.dropdown-content_link:hover {
```